What's new contribution guidelines update (#385)
grafana/writers-toolkit: docs/sources/contribute-documentation/contribute-release-notes/index.md

When you’re ready to add a What’s new entry, complete the following steps:
1. Fill out the fields:
  * FEATURE NAME - Short headline for the improvement. For example, “Grafana OnCall integration for Alerting.”
  * DATE - Date and time (UTC) that you want this note published. If you’ve opened a review PR, it will need to be merged before the date/time. If you enter a date that has passed, the note will publish immediately with that date. 
  * CONTACT - First and last name. The contents of this field are not publicly viewable. 
  * TAGS (OPTIONAL) - Select category tags that users can use to filter their view. Select as many as apply. 
  * CLOUD AVAILABILITY - Select the stage of the feature’s cloud release.
  * CLOUD OFFERING - Select which account types have access to the feature. 
  * SELF-MANAGED EDITIONS - Select the on-premises offering(s) where this feature is available.
  * BODY - Include an overview of the feature and the problem it solves. If you want to view some best practices around what to write here, see Writing guidelines for what’s new below.

>   Add any images and a link to your public Youtube video here. 

Author: ?

    - **GRAFANA URL PATH (OPTIONAL)** - URL path to the feature in a Grafana Cloud stack. For example, /alerting/notifications.
    - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements.

1. Request a review (optional).
   Select this option to open a PR in the website repository for review.
1. Click **Save**. When the note is saved, you can select **In review** from the **Status:** dropdown menu.
   The review PR will appear in the website repository. Note that the Documentation Team does not automatically review these requests; teams that create What’s new entries are responsible for determining their own review process. However, there are two weekly Office Hours meetings offered by the Documentation Team that you’re welcome to attend for guidance and assistance:

   - Docs squad office hours (early)
   - Docs squad office hours (late)
     If you open a PR, ensure that it’s merged in time for your feature release date.

1. Preview your entry and ensure that it looks the way you want it to.

>   For Grafana versioned releases, the content will be collected by [someone] and published in the versioned What’s new at a later date.

Reviewer: I'll look at this more closely on Monday, but we need to rewrite this section to align with the UI. You have to click **Save** to see the **Status** and **Publish** drop-downs. Then we'll need to add descriptions of the different actions available in those drop-downs.

> 

Reviewer: I'll look at this more closely on Monday, but we should add here descriptions of the different actions available in the Status and Publish drop-downs.

    1. Merge your PR in time for your feature release date.
    
       Merging your PR ensures your entry is published on the date you entered and it automatically updates the status of your entry in the CMS.

1. To publish your entry from the CMS, follow these steps:
    
      1. In the **Status** drop-down, click **Ready**.
      1. In the **Publish** drop-down, click **Publish now**. The entry will appear in [What's new in Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/) on the date you entered.

For Grafana versioned releases, the content entered in the CMS will be collected by [someone] and published in the versioned What’s new at a later date. Refer to [Creating the self-managed/versioned release notes](#creating-the-versioned-release-notes)

### Edit What's new entries

Whether your entry is published or not, it's always best to use the CMS to make any changes.

If your entry is published in both _What's new in Cloud_ and _What's new in Grafana_, or it's after the cut-off date for a versioned release, update the CMS and then reach out to the person who's creating the versioned release notes.

> ### Creating the versioned release notes

Reviewer: ```suggestion
### Create the versioned release notes
```

   - **DOCUMENTATION URL (OPTIONAL)** - URL to the public documentation for this feature.
   - **GRAFANA URL PATH (OPTIONAL)** - URL path to the feature in a Grafana Cloud stack. For example, `/alerting/notifications`.
   - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements.
   - **ENABLEMENT VIDEO (OPTIONAL)** - Link to the video used for enablement.

1. Click **Save**. The entry is now in **Draft** status.
1. If your entry is ready to publish, proceed to step 4. If your entry requires review, follow these steps:

   1. In the **Status** drop-down, select **In review.**

      A review PR is generated in the `grafana/website` repository. **The Documentation Team does not automatically review these requests; teams that create What’s new entries are responsible for determining their own review process.** However, there are two weekly Office Hours meetings offered by the Documentation Team that you’re welcome to attend for guidance and assistance:

>       - [Docs squad office hours (early)](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NmoxZ2w0NHBoazgxaW80cTN0MW82aG1xMmxfMjAyMzExMjJUMDkwMDAwWiBpc2FiZWwubWF0d2F3YW5hQGdyYWZhbmEuY29t&tmsrc=isabel.matwawana%40grafana.com&scp=ALL)
>       - [Docs squad office hours (late)](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NTI0MWZxYTYwY3FjYW5waWM1OWVjMDBkaWpfMjAyMzExMjJUMjEwMDAwWiBpc2FiZWwubWF0d2F3YW5hQGdyYWZhbmEuY29t&tmsrc=isabel.matwawana%40grafana.com&scp=ALL)
> 

Author: ```suggestion
      - [Docs squad office hours (early)](https://docs.google.com/document/d/1QaH9PwIZ_6-6Udhmy7Zhwme72ZZLqSTiX8HYFFi6AE4/edit#heading=h.9o53ccbx7xrw)
      - [Docs squad office hours (late)](https://docs.google.com/document/d/12XK3XYEZWU3uIPluq3sn5HSGqWHBHJkktqlyQZHj_Uw/edit#heading=h.9o53ccbx7xrw)

```
I think the links you provided @imatwawana will just link folks to their own calendar. Replaced with links to the agendas

Reviewer: Thanks for the catch. I noticed the links didn't look quite right later but forgot about it.

   1. In the **Status** drop-down, click **Ready**.
   1. In the **Publish** drop-down, click **Publish now**. The entry appears in [What's new in Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/) on the date you entered.

For Grafana versioned releases, the content entered in the CMS is published in the versioned What’s new at a later date. Refer to [Creating the versioned release notes](#create-the-versioned-release-notes)

### Edit What's new entries

Regardless of the status of your entry, it's always best to use the CMS to make any changes. To make edits, follow these steps:

1. Navigate to the [CMS](https://admin.grafana.com/content-admin/#/collections/whatsnew).
1. Update the fields that you need to change.
1. Click **Save**. The entry is now in **Draft** status.
1. Do one of the following:

   - If your entry is ready to publish, select **Ready** in the **Status** drop-down, and then **Publish now** in the **Publish** drop-down.
   - If your entry needs to be reviewed, select **In review** in the **Status** drop-down to open a review PR. For more information on managing review PRs, see step 3 in the [Create a What's new entry](#create-a-whats-new-entry).

> If your entry is already live in both _What's new in Cloud_ and _What's new in Grafana vxx_, or it's after the cut-off date for a versioned release, update the CMS and then reach out to the person responsible for creating the versioned release notes.

Reviewer 2: Words in backticks are not checked by the spell checker
```suggestion
If your entry is already live in both _What's new in Cloud_ and _What's new in Grafana `vMAJOR.MINOR.x`_, or it's after the cut-off date for a versioned release, update the CMS and then reach out to the person responsible for creating the versioned release notes.
```
